koswat/cost_report/io/summary/summary_costs/summary_costs_csv_fom_builder.py

```diff
@@ -108,6 +117,65 @@ def _get_summary_reinforcement_type_column_order(
             for _report in self.koswat_summary.locations_profile_report_list
         ]
 
+    def _get_total_cost(
```

Comment:
```suggestion
    def _get_total_infrastructure_cost(
```

Reply:
Actually it is the total cost. For infrastructure cost a different method is created.
The total cost is the sum of the measure cost and infrastructure cost.

Reply:
I would propose to keep it like this. The output format is similar to `_get_infrastructure_cost` and `_get_cost_per_selected_measure`.

Reply:
I still want to reduce the ambiguity, please change it as discussed via teams to a more descriptive option.